### DISCUSSION TITLE
Rouvy works

### DISCUSSION THREAD

**Discussion by chreds** [cagnulein/qdomyos-zwift#1669] [General]
I was able to use qz with Echelon ex4s (10") sideloaded with Rouvy. Just letting you know for: https://github.com/cagnulein/qdomyos-zwift/blob/master/docs/20_supported_devices_and_applications.md

Heartrate, cadence and resistance control came through as well.

**Comment by cagnulein** [comment]
wow that's a great news! are you on the qz facebook group too?

Il giorno lun 25 set 2023 alle 18:53 chreds ***@***.***> ha
scritto:

**Reply by chreds** [reply]
I am but haven't posted. I'm not very active on Facebook.

**Comment by chreds** [comment]
Only potential problem I've seen is sometimes qz advertises as a different device so I see duplicate sensors (since Rouvy keep track of the last one connected too). I'm guessing QZ changes how it advertises when you change settings to prevent caching issues or something? Or is there another reason and this is unexpected?

**Comment by cagnulein** [comment]
on mobile , this is a standard way to improve privacy. it's a rule that
apple and android added to their OS.
only a raspberry or a pc or a mac can have always the same mac address sent
to another device

Il giorno ven 29 set 2023 alle 16:45 chreds ***@***.***> ha
scritto:

**Reply by chreds** [reply]
Thanks. How close are you to selling the ESP32 devices? That seems ideal then I don't need the Android bridge.

**Comment by cagnulein** [comment]
i quitted the idea, too many legal aspect involved.


Il giorno ven 29 set 2023 alle 17:43 chreds ***@***.***> ha
scritto:

**Reply by chreds** [reply]
Are you still going to maintain the esp32 branch and just not sell the device. I'll probably just get a raspberry I guess instead as you're more likely to have more users on that and continue support there.

**Comment by cagnulein** [comment]
yes the esp32 code is here on github, it's quite simple after all
yes the raspberry would be the perfect choice

Il giorno ven 29 set 2023 alle 17:46 chreds ***@***.***> ha
scritto:
